src/styles/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
```

Comment:
Add a padding to product-card
`padding: 32px 16px 16px` and remove inline margins from all of the elements below